### ISSUE TITLE
[Bug] Pip install openbb fails with ResolutionTooDeep error

### ISSUE THREAD

**Issue by adrianmck1** [OpenBB-finance/OpenBBTerminal#5483]
I am trying to install the OpenBB SDK on Windows 10. I am following the [instructions ](https://docs.openbb.co/terminal/installation/pypi) on the OpenBB website. I was able to create the new environment, but when I try to use pip to install the openbb package as per the instructions (pip install openbb --no_cache-dir), it goes through the motions but fails after a very long time (eg. 2 hours) with a ResolutionTooDeep error. 
This is after using the conda-3-10.yaml file to create the environment.

Not sure where to go from here.

**To Reproduce**
conda activate obb
pip install openbb --no-cache-dir

**Screenshots**
I've captured the full output from pip below, but here's the end of it:

![image](https://github.com/OpenBB-finance/OpenBBTerminal/assets/122870179/21f8e0cb-b682-4485-aabe-d265b7e155ed)

**Desktop**
Windows 10
Python 3.10.8

**Additional context**
The output from pip is in the attachment.

[pip_install_openbb_output.txt](https://github.com/OpenBB-finance/OpenBBTerminal/files/12739145/pip_install_openbb_output.txt)

**Comment by jmaslek** [comment]
This is new to me.  Since it seems to be failing on yfinance, can you only do `pip install yfinance==0.2.28`  then retry the installation?

**Comment by deeleeramone** [comment]
Can you please share the output of `conda info`?

**Comment by deeleeramone** [comment]
@jmaslek, I think I found the issue:

![Screenshot 2023-09-27 134516](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/050cb676-932f-4ab6-aef9-73a95cc7dc85)


Looks like `financedatabase` is hijacking some dependencies with Pandas extras. Might need to do `--no-deps` on that.

**Comment by NervousNullPtr** [comment]
Installation also takes a crazy long time (1h35m) before it fails on the same item.

Linux User.

**Comment by deeleeramone** [comment]
> Installation also takes a crazy long time (1h35m) before it fails on the same item.

Can you try, `pip install openbb-nightly[all]`?

**Comment by Kopfstein** [comment]
> > Installation also takes a crazy long time (1h35m) before it fails on the same item.
> 
> Can you try, `pip install openbb-nightly[all]`?

Had a similar issue with `pip install openbb==3.2.2` running forever / not completing. Using the nightly worked as a quick fix as of today (2023-01-01).

**Comment by adrianmck1** [comment]
I also tried installing the nightly and it failed for me with an issue "building wheels for bt". I have attached the end of the output in case that helps.

[pip_install_openbb_nightly_fail.txt](https://github.com/OpenBB-finance/OpenBBTerminal/files/12780763/pip_install_openbb_nightly_fail.txt)

**Comment by deeleeramone** [comment]
> I also tried installing the nightly and it failed for me with an issue "building wheels for bt". I have attached the end of the output in case that helps.
> 
> [pip_install_openbb_nightly_fail.txt](https://github.com/OpenBB-finance/OpenBBTerminal/files/12780763/pip_install_openbb_nightly_fail.txt)

This is unrelated, and because you don't have Microsoft Build Tools installed, @adrianmck1.

```
 error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
 ```

![image](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/742d0e03-7cc3-46be-a1aa-684c6cf42c24)

**Comment by adrianmck1** [comment]
I had installed the build_tools.exe from the Microsoft website, but I didn't realise I had to specifically pick a workload. Maybe for thicko's like me, the installation page could be more explicit, as it doesn't seem to be called "C++ build tools" anymore, it's now called "Desktop development with C++"?

![image](https://github.com/OpenBB-finance/OpenBBTerminal/assets/122870179/32ee78d4-090d-4af8-8ed6-2b2fa41b52ce)

It's working for me now - thanks for your help!
